CHANGELOG.asciidoc

```diff
@@ -82,10 +84,16 @@ field. You can revert this change by configuring tags for the module and omittin
 - Fix config reload metrics (`libbeat.config.module.start/stops/running`). {pull}19168[19168]
 - Fix metrics hints builder to avoid wrong container metadata usage when port is not exposed {pull}18979[18979]
 - Server-side TLS config now validates that certificate and key settings are both specified {pull}19584[19584]
+- Fix terminating pod autodiscover issue. {pull}20084[20084]
+- Output errors when Kibana index pattern setup fails. {pull}20121[20121]
+- Fix issue in autodiscover that kept inputs stopped after config updates. {pull}20305[20305]
+- Add service resource in k8s cluster role. {pull}20546[20546]
+
```

Comment:
Nit: I don't know if the script that parses this handles new changes in the number of newlines between sections gracefully so it might be safest not to include any additional here.